src/shared/components/splash-screen/SplashScreen.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
The component is not properly tested in my opinion. Please consider the following use cases:

- `RNBootSplash.hide()` should be called when `hide === undefined`
- `RNBootSplash.hide()` should be called when `hide === true`
- `RNBootSplash.hide()` should be called with default `transitionDelay`
- `RNBootSplash.hide()` should be called with custom `transitionDelay`
- `RNBootSplash.hide()` should be called exactly once when `hide` or `autoHide` are `true`. It shouldn't be called anymore even if `hide` changes.